docs/developer/decision-records/2022-04-21-dpf-blob-data-transfer/README.md

```diff
@@ -0,0 +1,38 @@
+# Blob storage transfer
+
+ADR describing the blob storage transfer end to end flow between 2 participants.
```

Comment:
Are those two participants what is called `consumer` and `provider` in the following text?

Reply:
The 2 participants would be 2 companies part of the Dataspace. During the data transfer, one EDC instance has the role of Consumer (consuming data), and one has the role of Provider. 
One EDC instance can be both Consumer and Provider, its role depends of the data transfer. I didn't add the explanation to the doc because I think this is a fair assumption that somebody reading this doc knows the concept of Consumer/Provider as this is a concept used in EDC.